tmt/checks/avc.py

```diff
@@ -311,9 +321,15 @@ def create_final_report(
     else:
         outcome = ResultOutcome.ERROR
 
+    assert invocation.phase.step.workdir is not None  # narrow type
     report_filepath = _save_report(invocation, report, report_timestamp, append=True)
+    paths = [report_filepath.relative_to(invocation.phase.step.workdir)]
+
+    failures_path = save_check_failures(invocation, failures, logger) if failures else None
```

Comment:
Right, if `save_check_failures()` stops returning `Optional[Path]`, this would be also much simpler and cleaner:

```python
    paths = [
        report_filepath.relative_to(invocation.phase.step.workdir)
        save_check_failures(invocation, failures, logger)
    ]
```